app/scripts/controllers/transactions/index.test.js

```diff
@@ -1852,6 +1856,7 @@ describe('Transaction Controller', function () {
             token_standard: TokenStandard.none,
             device_model: 'N/A',
             transaction_speed_up: false,
+            ui_customizations: false,
```

Comment:
I am thinking that if we are passing `ui_customizations` we should return null rather than false so that we are not introducing a new data type